app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/contactDetails.scala.html

```diff
@@ -0,0 +1,62 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, Contact}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+
+@(c: Case)(implicit  messages: Messages)
```

Comment:
Can we make sure the relevant View tests are moved too if you haven't already :)

Reply:
Will do

Reply:
I also found a 3rd place were we had a 'contact details' section in line in the template so I've pulled that out too